### DISCUSSION TITLE
Dumb question: how do I update paru

### DISCUSSION THREAD

**Discussion by fritzrehde** [Morganamilo/paru#883] [Q&A]
I want to switch from `yay` to `paru`, so I went to the github readme and did
```
sudo pacman -S --needed base-devel
git clone https://aur.archlinux.org/paru.git
cd paru
makepkg -si
```
This built the rust project locally, instead of installing just a binary. Can I, alternatively, get a `paru` binary from the AUR using `yay`, and then update the `paru` from the AUR using `paru` in the future?
Because I am not installing `paru` through the AUR, will it still automatically update when I update all AUR/pacman packages?

**Comment by Morganamilo** [comment]
The AUR is source based. Yay also builds locally. There's paru-bin just like yay-bin for prebuild binaries.
